ansible/roles/set-repositories/tasks/rhn-repos.yml

```diff
@@ -34,6 +35,10 @@
     org_id: "{{ rhel_subscription_org_id }}"
     pool_ids: "{{ rhsm_pool_ids }}"
     auto_attach: false
+  ignore_errors: true  # Because of SCA
+
+- name: rhn | Enable RHSM to manage repositories
+  command: subscription-manager config --rhsm.manage_repos=1
```

Comment:
shouldn't this be parameterized?
Other configs using this role will now have rhsm enabled?

Reply:
I added this task because the next one was failing, as the RHSM was not configured to manage the repositories (during development...):

![image](https://github.com/user-attachments/assets/f3e16202-7472-41ec-a784-2cd5ed3292fa)

Will it work correctly if I remove this configuration?